acme.sh

```diff
@@ -4538,7 +4538,12 @@ list() {
         _initpath "$d" "$_isEcc"
         if [ -f "$DOMAIN_CONF" ]; then
           . "$DOMAIN_CONF"
-          printf "%s\n" "$Le_Domain${_sep}\"$Le_Keylength\"${_sep}$Le_Alt${_sep}$Le_CertCreateTimeStr${_sep}$Le_NextRenewTimeStr"
+          if [ -z "$Le_Keylength" ]; then
```

Comment:
don't need so complicated.
just one line:

```
printf "%s\n" "$Le_Domain${_sep}\"${Le_Keylength:-$DEFAULT_DOMAIN_KEY_LENGTH}\"${_sep}$Le_Alt${_sep}$Le_CertCreateTimeStr${_sep}$Le_NextRenewTimeStr"
```

Reply:
Thanks! I tested it out, and it doesn't have quite the same functionality (no square brackets for default values). Do you ever intent to merge this pull request, using my original changes or the ones you suggested?

Reply:
your original code  is too overkill.